toolchain/llvm/llvm-compiler.in

```diff
@@ -17,7 +17,11 @@ fi
 
 if [ "$CONF" == "1" ]; then
     SKIP_OPT="-O0 -fno-lto"
+else
+    if [ "@ENABLE_CCACHE@" = "ON" ]; then
+        CCACHE="ccache @CCACHE_OPTIONS@"
```

Comment:
```suggestion
        CCACHE_ENV='CCACHE_DIR="@MINGW_INSTALL_PREFIX@/ccache" CCACHE_MAXSIZE="500M"  CACHE_DIRECT="true"'
        CCACHE="$CCACHE_ENV ccache @CCACHE_OPTIONS@"
```
I would prefer this way instead putting in exec

Reply:
Based on [this](https://github.com/ccache/ccache/blob/master/doc/MANUAL.adoc#configuration), you dont need to export the variables. So this is possible:

`ccache cache_dir="@MINGW_INSTALL_PREFIX@/ccache" max_size=500M gcc -c example.c`

Default value for `direct_mode ` is true, so dont need to pass it